src/events/google_cloud.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * @fileoverview Instrumentation for google cloud common library.
+ */
+const uuid4 = require('uuid4');
+const shimmer = require('shimmer');
+const tryRequire = require('../try_require.js');
+const utils = require('../utils.js');
+const tracer = require('../tracer.js');
+const serverlessEvent = require('../proto/event_pb.js');
+const eventInterface = require('../event.js');
+const errorCode = require('../proto/error_code_pb.js');
+
+const common = tryRequire('@google-cloud/common/');
+
+const uriSplitString = 'googleapis.com/';
```

Comment:
Const should be all caps i think (look at other modules

Reply:
K